Content.Server/Cloning/CloningSystem.cs

```diff
@@ -84,6 +84,14 @@ public bool TryCloning(EntityUid original, MapCoordinates? coords, ProtoId<Cloni
         return true;
     }
 
+    public override void CloneComponents(EntityUid original, EntityUid clone, ProtoId<CloningSettingsPrototype> settings)
+    {
+        if (!_prototype.TryIndex(settings, out var proto))
```

Comment:
```suggestion
        if (!_prototype.Resolve(settings, out var proto))
```
this should log an error if you use an invalid id